osert.rb

```diff
@@ -275,7 +275,7 @@ def sed(file, text_i, text_o)
     end
 
     if options[:osid]
-      osid = options[:osid]
+      osid = "OS-" + options[:osid]
```

Comment:
One could also do `--osid OS-12345`. `OS-` is a fixed prefix, so naturally hardcoding it would save typing 3 characters. In interactive mode, prefixing the prompt with `OS-` makes sense it's displayed on screen, so one knows it's already prefixed. But in argument mode one doesn't see it, so many people will provide their full OSID `OS-12345` that will result in an unexpected `OS-OS-12345`. I think adding this will be less intuitive. It would require to explain this option is prefixed on the help message. So I don't see the value of adding this.